test-zeekoe.py

```diff
@@ -171,13 +171,27 @@ def check_blockchain_maturity(url):
         level = get_blockchain_level(url)
 
 class TestScenario():
-    def __init__(self, cust_config, channel_name, customer_deposit, verbose):
+    def __init__(
+            self, 
+            cust_config, cust_db, 
+            dev_path,  
+            channel_name, customer_deposit, 
+            verbose
+        ):
         self.cust_config = cust_config
+        self.cust_db = cust_db
+        self.dev_path = dev_path
+        self.temp_path = f"{dev_path}/temp"
+        self.channel_path = f"{self.temp_path}/{channel_name}"
```

Comment:
Non-blocking: Could you use `path.join()` to concatenate paths? https://stackoverflow.com/questions/13944387/why-use-os-path-join-over-string-concatenation

Reply:
Done! Thanks for the link with the explanation too.